worker/src/enclave/api.rs

```diff
@@ -272,6 +274,22 @@ pub fn enclave_accept_pdex_accounts(
     Ok(())
 }
 
+pub fn enclave_run_db_thread(eid: sgx_enclave_id_t) -> SgxResult<()> {
+    let mut status = sgx_status_t::SGX_SUCCESS;
+
+    let result = unsafe { run_db_thread(eid, &mut status) };
```

Comment:
Do we need to have the function `run_db_tread` to be unsafe or could it encapsulate the unsafeiness?

Reply:
It has to be unsafe as it's an extern function
https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/5f55a65430e60a2785200571da75b2d8b3e67b79/worker/src/enclave/api.rs#L69